packages/cli/src/zosfiles/ZosFilesBase.handler.ts

```diff
@@ -39,33 +39,39 @@ export abstract class ZosFilesBaseHandler implements ICommandHandler {
      *
      * @returns {Promise<void>}
      */
-    public async process(commandParameters: IHandlerParameters) {
-        const sessCfg: ISession = ZosmfSession.createSessCfgFromArgs(
-            commandParameters.arguments
-        );
-        const sessCfgWithCreds = await ConnectionPropsForSessCfg.addPropsOrPrompt<ISession>(
-            sessCfg, commandParameters.arguments, {parms: commandParameters}
-        );
-
-        const session = new Session(sessCfgWithCreds);
-        const response = await this.processWithSession(commandParameters, session);
-
-        commandParameters.response.progress.endBar(); // end any progress bars
-        // Print out the response
-        if (response.commandResponse) {
-            commandParameters.response.console.log(response.commandResponse);
-        }
-
-        // Return as an object when using --response-format-json
-        commandParameters.response.data.setObj(response);
-
-        // Ensure error gets thrown if request was unsuccessful.
-        // Sometimes it is useful to delay throwing an error until the end of the handler is
-        // reached, for example the upload API needs to return an API response even when it fails.
```

Comment:
Not a request for changes but please be careful about removing comments - some other ones in this file probably aren't very meaningful like "print out the response" so I understand removing them, but more verbose comments like this one may contain valuable information.